Library/Formula/symlinks.rb

```diff
@@ -0,0 +1,23 @@
+class Symlinks < Formula
+  desc "scan/change symbolic links"
+  homepage "http://www.ibiblio.org/pub/Linux/utils/file/symlinks.lsm"
```

Comment:
Nonetheless I believe we handle the situation more explicitly by passing `#{MacOS.sdk_path}/usr/include/` when there's no CLT present.